django_project/ford3/templates/provider.html

```diff
@@ -17,11 +17,21 @@
 		<div class="row no-gutters">
 			<div class='col-md-9'>
 				<h3>{{ provider.name }}</h3>
+			</div>
+			<div class='col-md-3' style="padding-left: 2.5rem;">
+				<a href="{% url 'edit-provider' provider_id=provider.id %}"
+				   class='edu-button edu-button-orange' style="margin: 0; padding: 0;">Edit Provider</a>
```

Comment:
same here do not apply style in the html. use bootstrap padding classes: `m-0 p-0`